ui/pages/confirmations/components/signature-request-original/index.scss

```diff
@@ -87,8 +85,6 @@
 
   &__notice,
   &__warning {
-    @include design-system.H6;
-
```

Comment:
Covered by the Text component here https://github.com/MetaMask/metamask-extension/pull/27076/files#diff-0b1e3dca811d174401e32d4e02e4f70ebf269526c26c58f788cb7c3f0ee28ce0R216

Reply:
Also removing unused styles as  `request-signature__notice` classname doesn't exist

![Screenshot 2024-09-15 at 12 22 27 PM](https://github.com/user-attachments/assets/9e22c692-3cec-48aa-b69f-4cf342c4c47c)